test/functionalTest/cases/722_unhardwire_format_cpr/notify_format_in_ngsi9_subscriptions.test

```diff
@@ -0,0 +1,944 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I will use cases/722_unhardwire_format_cpr for the test strictly related with CPrs. The "fixing subscription" test should be added to the corresponding case on subscriptions (I think we have a cases directory for NGSI9 subscriptions and a cases directory for NGSI10 subscriptsion).